lib/gitlab/client/repository_files.rb

```diff
@@ -51,9 +51,8 @@ def get_file(project, file_path, ref)
     # @param  [String] commit message.
     # @return [Gitlab::ObjectifiedHash]
     def create_file(project, path, branch, content, commit_message)
```

Comment:
Looking at the docs there seem to be a bunch of optional parameters we don't currently support.  Do you think we should add `options={}` and update the YARD to include the optional parameters defined in the Gitlab API doc for `Repository Files`?